Client/Ecosia/UI/Theme/EcosiaThemeManager.swift

```diff
@@ -79,10 +75,11 @@ public final class EcosiaThemeManager: ThemeManager, Notifiable {
     public func changeCurrentTheme(_ newTheme: ThemeType) {
         guard currentTheme.type != newTheme else { return }
         currentTheme = newThemeForType(newTheme)
+        updateLegacyThemeIfNeeded()
 
         // overwrite the user interface style on the window attached to our scene
         // once we have multiple scenes we need to update all of them
-        window?.overrideUserInterfaceStyle = currentTheme.type.getInterfaceStyle()
+        // window?.overrideUserInterfaceStyle = currentTheme.type.getInterfaceStyle()
```

Comment:
I guess since this is an Ecosia file we can just remove this commented code, right? Any reason to keep it?

Reply:
You are totally right on that! Will update